src/Components/EditProduct.tsx

```diff
@@ -1,207 +1,176 @@
-import React, { Dispatch, ReactNode, SetStateAction, useEffect, useState } from 'react'
-import LandingPage from './LandingPage'
-import { useParams } from 'react-router-dom'
-import { Toolbar } from 'primereact/toolbar';
-import { DataTable, DataTableFilterMeta, } from 'primereact/datatable';
-import { Column } from 'primereact/column';
-import { Dropdown, DropdownChangeEvent,} from 'primereact/dropdown';
-import { Panel } from 'primereact/panel';
-import { useNavigate } from 'react-router-dom';
-import { Fieldset } from 'primereact/fieldset';
-import { InputText } from 'primereact/inputtext';
-import { Button } from 'primereact/button';
-import AddProduct from './AddProduct';
-import TopContent from './TopContent';
-import { SplitButton } from 'primereact/splitbutton';
+import React, { Dispatch, SetStateAction, useEffect, useState } from "react";
+import LandingPage from "../Pages/LandingPage";
+import { useNavigate, useParams } from "react-router-dom";
+import { Toolbar } from "primereact/toolbar";
+import { DataTable } from "primereact/datatable";
+import { Column } from "primereact/column";
+import { Panel } from "primereact/panel";
+import { Fieldset } from "primereact/fieldset";
+import { Button } from "primereact/button";
+import AddProduct from "./AddProduct";
+import TopContent from "./TopContent";
+import { SplitButton } from "primereact/splitbutton";
+import { prod } from "../data";
 
 
-interface VisibilityProps {
-  setVisible?: Dispatch<SetStateAction<boolean>>;
-  visible?: boolean
-}
 interface Product {
```

Comment:
we can make new folder /src/models/model_name (Product here) to more organize your code.